docs/README.ubuntu

```diff
@@ -113,6 +113,16 @@ We supply a Makefile in lib/taglib to make it easy to install into /usr/local.
 or use prepackaged from the Kodi PPA.
 
     $ sudo apt-get install libtag1-dev
+	
+For >= 14.04
```

Comment:
Well not really utopic isnt there so only for v14.04 and 15.04

I'll add this note for now.